pkg/kubernetes/resources/resources.go

```diff
@@ -0,0 +1,32 @@
+package resources
+
+import (
+	"context"
+
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
```

Comment:
Relying on `dynamic` client here allows us to read the CRDs without importing the whole API.